builtin.go

```diff
@@ -387,8 +373,11 @@ func handleISUPPORT(c *Client, e Event) {
 	}
 
 	c.state.Lock()
-	// Skip the first parameter, as it's our nickname.
-	for i := 1; i < len(e.Params); i++ {
+	// Skip the first parameter, as it's our nickname, and the last, as it's the doc.
+	for i := 1; i < len(e.Params)-1; i++ {
+		if strings.Contains(e.Params[i], " ") {
+			continue
+		}
```

Comment:
Unless I'm missing something, only the last parameter can contain spaces, so this feels superfluous.

Reply:
Yeah, I added the check for spaces before I had the for loop just skip the last item in the array. Forgot to remove, good catch.